### ISSUE TITLE
[Authentication code] Generated code container missing role

### ISSUE THREAD

**Issue by svinkle** [CovidShield/portal#16]
## Issue summary

The text node which houses the generated code is marked up as a `div`. This provides no semantic meaning or structure to indicate the importance of this content. Additional issues may arise as a result of the DOM order:

1. List instruction
2. Generate code control
3. _User navigates forward… "Where is the code? 🤔"_

<img width="586" alt="Screen Shot 2020-05-18 at 11 49 40 AM" src="https://user-images.githubusercontent.com/1392632/82233238-b6d85480-98fd-11ea-8e90-58263f4a38ea.png">

#### HTML

```html
<div id="code-result" class="code-generator__banner" …>
  <div data-code-result="" aria-live="polite"></div>
  <!-- … -->
</div>
```

## Steps to reproduce

1. Go to https://portal-rails.myshopify.io/ with with any supported browser
2. Login with `admin`/`password`
3. With an active screen reader, navigate through the page content via virtual cursor

### Expected

- Generated code region to be conveyed.

### Actual

- Generated code region is skipped over.

## Recommendation

1. On the `#code-result` container, add the `role` attribute, setting its value to `region`. This sets the element as a landmark region.
2. Add the `aria-label` attribute, settings its value to `Generated code`. This provides an accessible name to the new landmark.
3. On the `data-code-result` container, swap the non semantic `div` element for `strong` to indicate the importance of this content.

#### HTML

```html
<div id="code-result" class="code-generator__banner" role="region" aria-label="Generated code" …>
  <strong data-code-result="" aria-live="polite"></strong>
  <!-- … -->
</div>
```

## Specifications

- **Component:** Authentication code
- **WCAG Principle:** Robust
- **WCAG SC:** 4.1.2 Name, Role, Value
- **Severity:** High
- **Effort:** Low
